app/scripts/metamask-controller.js

```diff
@@ -3036,4 +3057,16 @@ export default class MetamaskController extends EventEmitter {
   setLocked() {
     return this.keyringController.setLocked();
   }
+
+  //qr hardware related methods
+
+  initialQRHardwareState() {
```

Comment:
Instead of putting this here, we could probably add this to the `AppState` controller.